src/app/containers/Bulletin/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import pathOr from 'ramda/src/pathOr';
+import { shape, oneOfType } from 'prop-types';
+import Image from '@bbc/psammead-image';
+import Bulletin from '@bbc/psammead-bulletin';
+import { tvBulletinItem, radioBulletinItem } from '#models/propTypes/bulletin';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const BulletinContainer = ({ item }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+
+  const contentType = pathOr(null, ['contentType'], item);
+  const isLive = pathOr(null, ['isLive'], item);
+  const headline = pathOr(null, ['name'], item);
+  const summary = pathOr(null, ['summary'], item);
+  const ctaLink = pathOr(null, ['uri'], item);
+  const imageSrc = pathOr(null, ['indexImage', 'href'], item);
+  const imageAlt = pathOr(null, ['indexImage', 'altText'], item);
+  const bulletinImage = <Image src={imageSrc} alt={imageAlt} />;
+
+  const ctaText = contentType === 'TVBulletin' ? 'Watch' : 'Listen';
```

Comment:
Do we need to add translations for these rather than hardcode english? (I think Jon mentioned trying to get hold of these in standup)

Reply:
I have created this Issue to address that. https://github.com/bbc/simorgh/issues/4863